src/main/java/figure/FigureMain.java

```diff
@@ -0,0 +1,10 @@
+package figure;
+
+public class FigureMain extends FigureSupplier {
+    public static void main(String[] args) {
+        Figure[] figures = createRandomArrayOfFigures(5);
```

Comment:
5 is a magic number https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names